src/pages/Home.jsx

```diff
@@ -7,86 +7,84 @@ function Home () {
     <div>
       {/* Mobile Version */}
       <div className='lg:hidden'>
-        <div className='flex bg-white py-2.5'>
-          <p className='font-normal font-header text-xl ml-auto mx-2'>
+        <div className='flex justify-center text-center gap-2 bg-white py-8'>
+          <p className='font-normal font-header text-xl'>
             Tap to call!
           </p>
-          <p className='font-normal font-header text-xl text-green-700 mx-2'>
+          <p className='font-normal font-header text-xl text-green-700'>
             123.456.7890
           </p>
         </div>
-        <div className="p-6 bg-white bg-[url('/public/images/hero.png')] bg-no-repeat bg-cover">
-          <div className='flex flex-col items-center justify-center p-6 bg-white bg-opacity-80 rounded-2xl'>
-            <div className='py-5 px-16 gap-2.5'>
-              <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
-              <hr className='w-[211px] h-px mx-auto mt-4 bg-black border-0 rounded'></hr>
-            </div>
+        <div className="p-6 bg-white bg-[url('/public/images/heromobile.png')] bg-no-repeat bg-cover">
+          <div className='flex flex-col items-center justify-center p-6 bg-white bg-opacity-80 my-10'>
             <div>
-              <div className='w-[358px] h-[80px] px-8'>
-                <h2 className='font-normal italic text-center text-3xl text-zinc-500'>
-                  Detroit&apos;s Finest Lawn Care
-                </h2>
+              <div className='px-16 gap-2.5 pt-10'>
+                <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
+              </div>
+              <div>
+                <div className='px-8'>
+                  <h2 className='font-normal italic text-center text-3xl text-zinc-500'>
```

Comment:
Question (nb, can refactor later): Why use `BaseHeading` in some places and heading tags in others?